Src/WitsmlExplorer.Frontend/components/Modals/ConfirmModal.tsx

```diff
@@ -19,6 +20,7 @@ const ConfirmModal = (props: ConfirmProps): React.ReactElement => {
       onSubmit={props.onConfirm}
       isLoading={false}
       confirmText={props.confirmText ?? "Yes"}
+      cancelText={props.cancelText ?? null}
```

Comment:
I think there is no need to add "?? null" as props.cancelText will be null/undefined anyways?

Reply:
done